Traceback (most recent call last):
  File "./train-l2n40.py", line 6, in <module>
    calc.train('data.db')
  File "/home-research/jboes/boes-python/neural/bp/__init__.py", line 707, in train
    gtol=10. ** -500.)
  File "/opt/kitchingroup/CANOPY/Canopy_64bit/User/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 777, in fmin_bfgs
    res = _minimize_bfgs(f, x0, args, fprime, callback=callback, **opts)
  File "/opt/kitchingroup/CANOPY/Canopy_64bit/User/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 850, in _minimize_bfgs
    old_fval, old_old_fval)
  File "/opt/kitchingroup/CANOPY/Canopy_64bit/User/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 690, in _line_search_wolfe12
    old_fval, old_old_fval)
  File "/opt/kitchingroup/CANOPY/Canopy_64bit/User/lib/python2.7/site-packages/scipy/optimize/linesearch.py", line 263, in line_search_wolfe2
    derphi0, c1, c2, amax)
  File "/opt/kitchingroup/CANOPY/Canopy_64bit/User/lib/python2.7/site-packages/scipy/optimize/linesearch.py", line 363, in scalar_search_wolfe2
    phi0, derphi0, c1, c2)
  File "/opt/kitchingroup/CANOPY/Canopy_64bit/User/lib/python2.7/site-packages/scipy/optimize/linesearch.py", line 498, in _zoom
    phi_aj = phi(a_j)
  File "/opt/kitchingroup/CANOPY/Canopy_64bit/User/lib/python2.7/site-packages/scipy/optimize/linesearch.py", line 239, in phi
    return f(xk + alpha * pk, *args)
  File "/opt/kitchingroup/CANOPY/Canopy_64bit/User/lib/python2.7/site-packages/scipy/optimize/optimize.py", line 281, in function_wrapper
    return function(*(wrapper_args + args))
  File "/home-research/jboes/boes-python/neural/bp/__init__.py", line 1238, in f
    self.force_rmse))
  File "/home-research/jboes/boes-python/neural/utilities.py", line 97, in __call__
    self._f.flush()
IOError: [Errno 2] No such file or directory
